cfme/configure/access_control.py

```diff
@@ -103,55 +154,74 @@ def update(self, updates):
+        view.configuration.item_select('Copy this User to a new User')
+        view = self.create_view(AddUserView)
+        new_user = User(name="{}copy".format(self.name),
                         credential=Credential(principal='redhat', secret='redhat'))
         change_stored_password()
-        fill(self.user_form, {'name_txt': new_user.name,
-                              'userid_txt': new_user.credential.principal,
-                              'password_txt': new_user.credential.secret,
-                              'password_verify_txt': new_user.credential.verify_secret},
-             action=form_buttons.add)
-        flash.assert_success_message('User "{}" was saved'.format(new_user.name))
+        view.fill({
+            'name_txt': new_user.name,
+            'userid_txt': new_user.credential.principal,
+            'password_txt': new_user.credential.secret,
+            'password_verify_txt': new_user.credential.verify_secret
+        })
+        view.add_button.click()
+        view = self.create_view(AllUserView)
+        view.flash.assert_success_message('User "{}" was saved'.format(new_user.name))
+        assert view.is_displayed
         return new_user
 
     def delete(self):
-        navigate_to(self, 'Details')
-        tb.select('Configuration', 'Delete this User', invokes_alert=True)
-        sel.handle_alert()
-        flash.assert_success_message('EVM User "{}": Delete successful'.format(self.name))
+        view = navigate_to(self, 'Details')
+        view.configuration.item_select('Delete this User', handle_alert=True)
+        view = self.create_view(AllUserView)
+        view.flash.assert_success_message('EVM User "{}": Delete successful'.format(self.name))
+        assert view.is_displayed
 
     def edit_tags(self, tag, value):
-        navigate_to(self, 'Details')
-        pol_btn("Edit 'My Company' Tags for this User", invokes_alert=True)
-        fill(edit_tags_form, {'select_tag': tag,
-                              'select_value': value},
-             action=form_buttons.save)
-        flash.assert_success_message('Tag edits were successfully saved')
+        view = navigate_to(self, 'EditTags')
+        view.fill({'select_tag': tag,
+                   'select_value': value})
+        view.save_button.click()
+        view = self.create_view(DetailsUserView)
+        view.flash.assert_success_message('Tag edits were successfully saved')
+        assert view.is_displayed
 
     def remove_tag(self, tag, value):
-        navigate_to(self, 'Details')
-        pol_btn("Edit 'My Company' Tags for this User", invokes_alert=True)
-        row = tag_table.find_row_by_cells({'category': tag, 'assigned_value': value},
-                                          partial_check=True)
-        sel.click(row[0])
-        form_buttons.save()
-        flash.assert_success_message('Tag edits were successfully saved')
+        view = navigate_to(self, 'EditTags')
+        row = view.tag_table.row((1, tag), (2, value))
```

Comment:
.row always return a single item
Also updated indexes to name column, except row[0].click() as this column doesn't have name